pkgs/applications/version-management/srcml/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, cmake, libxml2, libxslt, boost, libarchive, python, antlr }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  pname = "srcml";
```

Comment:
I use this pattern everywhere, and hope that there will semantic version names soon - at which point it will. Is it okay to leave as is?